solr/core/src/test/org/apache/solr/util/TestCircuitBreaker.java

```diff
@@ -157,6 +177,60 @@ public void testBuildingMemoryPressure() {
     }
   }
 
+  public void testFakeCPUCircuitBreaker() {
+    ExecutorService executor = ExecutorUtil.newMDCAwareCachedThreadPool(
```

Comment:
Better practice to declare this right before the `try` block, without intervening declarations.